src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureBase {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int firstSide, int secondSide, int height) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide + secondSide) / 2 * height;
```

Comment:
To ensure precision in floating-point calculations and avoid integer division, cast one operand to double before performing the division or use `2.0` instead of `2`.